src/store/actionTypes.ts

```diff
@@ -13,6 +14,13 @@ export enum ActionTypes {
 export type ActionsPayload = {
   [ActionTypes.RESET_APP_NAME]: undefined;
   [ActionTypes.SET_APP_NAME]: string;
+  [ActionTypes.SET_CURRENT_USER]: {
```

Comment:
```suggestion
  [ActionTypes.SET_CURRENT_USER]: User
```

Please extract into separate type and make it reusable (it might potentially be used somewhere else)